trappy/utils.py

```diff
@@ -102,3 +105,55 @@ def handle_duplicate_index(data,
             dup_index_left += 1
 
     return data.reindex(new_index)
+
+def merge_dfs(pr_df, sec_df, pivot):
```

Comment:
Please add a doctsring for this method and explain with an example in the documentation. 

(A short excerpt like the one you have in the gist.)